README.md

```diff
@@ -1,7 +1,10 @@
+![alt text](https://www.devitoproject.org/images/banner-devito.jpg)
```

Comment:
Actually, I'm not sure we want the banner here ?

Reply:
Whatever you want, np...

Reply:
I'd say let's drop it

Reply:
dropped